src/bindings/capi/thorvg_capi.h

```diff
@@ -560,19 +560,20 @@ TVG_API Tvg_Canvas* tvg_wgcanvas_create(void);
 /*!
 * \brief Sets the drawing target for the rasterization.
 *
-* \param[in] instance WGPUInstance, context for all other wgpu objects.
-* \param[in] surface WGPUSurface, handle to a presentable surface.
-* \param[in] w The width of the surface.
-* \param[in] h The height of the surface.
-* \param[in] device WGPUDevice, a desired handle for the wgpu device. If it is @c nullptr, ThorVG will assign an appropriate device internally.
+* @param[in] device WGPUDevice, a desired handle for the wgpu device. If it is @c nullptr, ThorVG will assign an appropriate device internally.
+* @param[in] instance WGPUInstance, context for all other wgpu objects.
+* @param[in] surface or texture either WGPUSurface or WGPUTexture, serving as handles to a presentable surface or texture.
```

Comment:
target Either WGPUSurface or WGPUTexture, serving as handles to a presentable surface or texture.